src/scripts/components/foundations/forms/Input.js

```diff
@@ -9,10 +9,16 @@ const AutofillFix = props => keyframes`
 
 export const StyleFieldText = css`
   background-color: transparent;
+  color: ${props => props.theme.colors.TextField.color};
   font-family: ${props => props.theme.fonts.family}, sans-serif;
   font-size: ${props => props.theme.fonts.form.input};
   font-weight: ${props => props.theme.fonts.weight.regular};
   width: 100%;
+
+  &:-webkit-autofill {
+    animation-name: ${AutofillFix};
+    animation-fill-mode: both;
+  }
```

Comment:
What about other browsers?

Reply:
Ah I see this was just moved. But what about other browsers?

Reply:
this is only for the yellow background from webkit